folding-schemes/src/folding/nova/nifs/mova_matrix.rs

```diff
@@ -87,39 +87,40 @@ impl<C: Curve> Witness<C> {
     /// * `self` - Witness instance to be committed.
     /// * `params` - Commitment scheme parameters.
     /// * `rE` - Random evaluation point for the committed instance.
-    pub fn commit<CS: CommitmentScheme<C, H>, const H: bool>(
+    pub fn commit<CS: CommitmentScheme<C, H> + NethermindCommitmentScheme<C>, const H: bool>(
```

Comment:
`NethermindCommitmentScheme` is defined as a subtrait of `CommitmentScheme`, so I believe `NethermindCommitmentScheme<C, H>` is enough and the concatenation of traits is not needed